.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build:
     name: Build, lint, and test
-    runs-on: ubuntu-latest
+    runs-on: macos-15
```

Comment:
Potential alternative if this doesn't work well: https://playwright.dev/docs/ci#workers